sprint.rst

```diff
@@ -40,3 +40,18 @@ OpenSourceProject
 
 <any additional info>
 
+Live Coding in Python
+-----------------
```

Comment:
The underline should be the same length as the title. So you'll need to add 4 more dashes here.
Once fixed, the build should pass :)